ui/pages/confirmations/components/confirm/footer/footer.tsx

```diff
@@ -85,16 +70,16 @@ function ConfirmButton({
         startIconName={
           hasDangerAlerts ? getIconName(hasUnconfirmedDangerAlerts) : undefined
         }
-        onClick={hasDangerFieldAlerts ? handleOpenConfirmModal : onSubmit}
-        danger={hasDangerAlerts ? true : isDangerButton}
+        onClick={hasDangerAlerts ? handleOpenConfirmModal : onSubmit}
+        danger={hasDangerAlerts}
         size={ButtonSize.Lg}
         disabled={hasUnconfirmedDangerAlerts ? false : disabled}
       >
         {hasUnconfirmedDangerAlerts ? t('reviewAlerts') : t('confirm')}
```

Comment:
After our discussion regarding the designs for [the MVP](https://www.figma.com/design/yUczlH2HbHaHsYijiONKx0/Warnings%2C-Alerts-and-Flags?node-id=2113%3A40681&t=Ac1b3BuZOUUrsP54-1) , I wanted to confirm a few details.
Once we receive a general alert from Blockaid, the button will be labeled `Review Alerts`. Clicking this button will open a friction modal where the user can acknowledge the alert and proceed to confirm the action.

Is that correct?

Reply:
Yes, that is how it will work.